src/openforms/formio/formatters/formio.py

```diff
@@ -121,13 +121,17 @@ def format(self, component: SelectBoxesComponent, value: dict[str, bool]) -> str
             # For the html output, wrap the values in li tags and put it inside an ul tag.
             # The selectboxes formatter handles all values at the same time,
             # so handle the full html formatting here.
-            return format_html(
-                "<ul>{values}</ul>",
-                values=format_html_join(
-                    "",
-                    "<li>{}</li>",
-                    ((selected_label,) for selected_label in selected_labels),
-                ),
+            return (
+                format_html(
+                    "<ul>{values}</ul>",
+                    values=format_html_join(
+                        "",
+                        "<li>{}</li>",
+                        ((selected_label,) for selected_label in selected_labels),
+                    ),
+                )
+                if len(selected_labels) > 0
+                else ""
```

Comment:
codewise I think a guard clause would be more readable:

```py
if not selected_labels:
    return ""
return format_html(...)
```
this also makes it more easy to see where we're generating (potentially unsafe) HTML that needs to escape inputs.